src/sources/sources.jl

```diff
@@ -58,7 +58,10 @@ end
 
 # error message to show when backend is not loaded
 function Base.showerror(io::IO, e::BackendException)
-    print(io, "`Rasters.jl` requires backends to be loaded externally as of version 0.8. Run `import $(e.backend)` to fix this error.")
+    printstyled(io, "Rasters.jl"; underline = true)
+    printstyled(io, " requires backends to be loaded externally as of version 0.8.  Run ")
```

Comment:
```suggestion
    printstyled(io, " requires backends to be loaded manually.  Run ")
```
I think we can cut the version 0.8 part now